packages/react-tooltip/src/stories/TooltipCustomMount.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+import { Tooltip, TooltipProps } from '../Tooltip';
+import { Button } from '@fluentui/react-button';
+import { SlideTextRegular } from '@fluentui/react-icons';
+
+export const CustomMount = (props: Partial<TooltipProps>) => {
+  const divRef = React.useRef<HTMLDivElement>(null);
+  return (
+    <div ref={divRef}>
+      <Tooltip
+        mountNode={divRef.current ? divRef.current : undefined}
```

Comment:
This example is not correct as during first render `divRef.current` will be always `null` as refs resolved after a rendering cycle.

Please use `[ref, setRef] = React.useState()` instead.

Reply:
fix this to match the typings proposed in #22153